upnp/src/api/upnpapi.c

```diff
@@ -4162,6 +4162,9 @@ int UpnpAddVirtualDir(const char *newDirName)
     while( pCurVirtualDir != NULL ) {
 	/* already has this entry */
 	if( strcmp( pCurVirtualDir->dirName, dirName ) == 0 ) {
+            if (oldcookie != NULL)
+                *oldcookie = pCurVirtualDir->cookie;
+            pCurVirtualDir->cookie = cookie;
```

Comment:
these lines are indented using space while the surrounding lines use a mixture of tab and space :-\

Reply:
The indentation changes all over the place. I did try to to match mostly

Reply:
I know the code is still tab inconsistent, historically I have tried to make it consistent as I go through the files I change.

Just for your information, I think it is ok to change the whole file indentation to something sane whenever a patch for that file is coming ahead. Just make sure the white space patch comes separate.

Regards!